x-pack/filebeat/input/azureeventhub/input.go

```diff
@@ -235,3 +236,15 @@ func (a *azureInput) parseMultipleMessages(bMessage []byte) []string {
 	}
 	return messages
 }
+
+// Strip connection string to remove sensitive information
+// A connection string should look like this:
+// Endpoint=sb://dummynamespace.servicebus.windows.net/;SharedAccessKeyName=DummyAccessKeyName;SharedAccessKey=5dOntTRytoC24opYThisAsit3is2B+OGY1US/fuL3ly=
+// This code will remove everything after ';' so key information is stripped
+func stripConnectionString(c string) string {
+	if parts := strings.SplitN(c, ";", 2); len(parts) == 2 {
+		return parts[0]
+	}
+
+	return "(redacted)"
```

Comment:
Why returning `(redacted)` if there is nothing after the endpoint? Would it be ok to return `c`?

Reply:
This is my thinking here, perhaps I should add a comment about it:

They only way to pass credentials here is putting them on the connection string (@narph please confirm if this is correct.), so if for any reason `;` is not part of it there is something wrong going on, so I stay on the safe side.

Reply:
Oh ok, makes sense :+1:

Reply:
And +1 to add a comment mentioning that the connection string may contain the credentials.

Reply:
done b3635a7

Reply:
Thanks :+1: